scripts/zkai_native_attention_mlp_rmsnorm_label_policy_gate.py

```
@@ -0,0 +1,739 @@
+    try:
+        return sensitivity_gate.require_output_path(path)
+    except sensitivity_gate.RmsnormLabelSensitivityError as err:
+        raise LabelPolicyError(f"failed to normalize output path: {err}") from err
+
+
+def write_outputs(payload: dict[str, Any], json_path: pathlib.Path | None, tsv_path: pathlib.Path | None) -> None:
+    validate_payload(payload)
+    json_target = normalize_output_path(json_path) if json_path is not None else None
+    tsv_target = normalize_output_path(tsv_path) if tsv_path is not None else None
+    if json_target is not None and tsv_target is not None and json_target.resolve() == tsv_target.resolve():
+        raise LabelPolicyError(f"duplicate output destination: {json_target}")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)